askama_derive/src/generator.rs

```diff
@@ -1915,18 +2036,19 @@ where
     }
 }
 
-impl MapChain<'_, &str, LocalMeta> {
-    fn resolve(&self, name: &str) -> Option<String> {
-        let name = normalize_identifier(name);
-        self.get(&name).map(|meta| match &meta.refs {
+impl MapChain<'_, Cow<'_, str>, LocalMeta> {
+    fn resolve(&self, name: Cow<'_, str>) -> Option<String> {
```

Comment:
Could we avoid the caller having to wrap their `name` in `Cow`? Same for `resolver_or_self()`.

Reply:
Sure.